src/coordination/include/nuraft/coordinator_state_manager.hpp

```diff
@@ -60,6 +61,7 @@ class CoordinatorStateManager : public state_mgr {
   ptr<srv_config> my_srv_config_;
   ptr<cluster_config> cluster_config_;
   ptr<srv_state> saved_state_;
+  std::unique_ptr<kvstore::KVStore> kv_store_;
```

Comment:
I think you don't need unique_ptr to kvstore? just use value semantic

Reply:
The problem is you can't default construct it because that constructor is deleted. You can only construct it with path. But that way I can't check if the path is valid so I would like to leave it as it is

Reply:
I think no problem with this, I see many places where check isn't done. Or do the check before, in memgraph.cpp. I think it doesn't make sense to unique_ptr for that

Reply:
I will do check before